### DISCUSSION TITLE
Unattended Install

### DISCUSSION THREAD

**Discussion by EricDriussi** [searxng/searxng#1364] [Q&A] [answered]
I understand there are changes being made to the install scripts but are there any plans to add some sort of unattended install flag?
I'm trying to include this in an Ansible script and it would be really nice to have a command to tell the installer "just do your thing".

Cheers!

**Comment by return42** [comment, marked as answer]
``FORCE_TIMEOUT=0 ./utils/searxng.sh install all`` / see https://github.com/searxng/searxng/pull/1332

**Reply by EricDriussi** [reply]
Absolutely missed that, thanks a lot!
